src/rnp/rnpcfg.h

```diff
@@ -61,6 +61,8 @@
 #define CFG_CREATION "creation"         /* signature validity start */
 #define CFG_CIPHER "cipher"             /* symmetric encryption algorithm as string */
 #define CFG_HASH "hash"                 /* hash algorithm used, string like 'SHA1'*/
+#define CFG_S2K_ITER "s2k-iter"         /* number of S2K hash iterations to perform */
+#define CFG_S2K_MSEC "s2k-msec"         /* number of milliseconds S2K should target */
```

Comment:
I'm not seeing this option being read in the code.

Reply:
Fixed